app/assets/stylesheets/editor/_matches.scss

```diff
@@ -30,3 +30,8 @@
     color: $white;
   }
 }
+
+.hidden__item {
```

Comment:
This isn't quite how BEM works. `[b]lock__[e]lement--[m]odifier`, so in this case there is no `hidden` block. A better name might be `matches__hidden` or `matches__item--hidden` if the class is applied directly to the same element rather than in a separate span.